### PR TITLE
#5852 Buefy icon to neo icon

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [ ] Closes #<issue_number>
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖 Generated by Copilot at 191f617</samp>

This pull request adds, replaces, or updates various icons and tooltips in the UI components of the NFT gallery, using `NeoIcon` and `Font Awesome` libraries. It also adds or improves some features related to NFT minting, filtering, liking, and displaying. It affects the following files: `components/common/FrequentlyAsked.vue`, `components/common/IdentityForm.vue`, `components/explore/FilterMenuButton.vue`, `components/identity/module/IdentityPopoverHeader.vue`, `components/identity/module/IdentitySocial.vue`, `components/landing/CuratedList.vue`, `components/landing/EsCarbonless.vue`, `components/landing/EsSustainability.vue`, `components/Navbar.vue`, `components/navbar/MobileExpandableSection.vue`, `components/rmrk/Create/DropUpload.vue`, `components/rmrk/Create/SimpleMint.vue`, `components/rmrk/Gallery/EmotionList.vue`, `components/rmrk/Gallery/GalleryCard.vue`, `components/rmrk/Gallery/History.vue`, `components/rmrk/Gallery/Holder/Holder.vue`, `components/rmrk/Gallery/Layout.vue`, `components/TheFooter.vue`, `components/common/LoadPreviousPage.vue`, and `components/rmrk/Gallery/Item/Facts.vue`.

### <samp>🤖 Generated by Copilot at 191f617</samp>

> _We're adding icons to the pages, `NeoIcon` is the name_
> _We're using Font Awesome brands, to give them all some fame_
> _We're refactoring the logic, to make the code more clean_
> _We're heaving on the count of three, to make the website gleam_

### CODE REVIEW

**Comment by prachi00** [comment]
Also, does anyone know why deepsource says neoIcon is defined but isnt used, when it IS being used 
Screen Shot 2023-05-03 at 9 39 21 PM

File: components/identity/module/IdentityPopoverHeader.vue [hunk 1]
```diff
@@ -70,4 +72,10 @@ const toast = (message: string) => {
     }
   }
 }
+
+.is-link {
```
Comment (roiLeo) [inline components/identity/module/IdentityPopoverHeader.vue:76]: have you tried `has-text-link`?

File: components/rmrk/Gallery/History.vue [hunk 2]
```diff
@@ -15,8 +15,8 @@
             {{ $t('history.label') }}
           </p>
           <a class="card-header-icon">
-            <b-icon :icon="props.open ? 'chevron-up' : 'chevron-down'">
-            </b-icon>
+            <NeoIcon :icon="props.open ? 'chevron-up' : 'chevron-down'">
+            </NeoIcon>
```
Comment (roiLeo) [inline components/rmrk/Gallery/History.vue:19]:
```suggestion
            <NeoIcon :icon="props.open ? 'chevron-up' : 'chevron-down'" />
```

File: components/rmrk/Gallery/Holder/Holder.vue [hunk 3]
```diff
@@ -15,8 +15,8 @@
             {{ collapseTitleOption || $t('holders') }}
           </p>
           <a class="card-header-icon">
-            <b-icon :icon="props.open ? 'chevron-up' : 'chevron-down'">
-            </b-icon>
+            <NeoIcon :icon="props.open ? 'chevron-up' : 'chevron-down'">
+            </NeoIcon>
```
Comment (roiLeo) [inline components/rmrk/Gallery/Holder/Holder.vue:19]:
```suggestion
            <NeoIcon :icon="props.open ? 'chevron-up' : 'chevron-down'" />
```

File: components/rmrk/Gallery/PriceChart.vue [hunk 4]
```diff
@@ -13,7 +13,8 @@
           {{ $t('chart') }}
         </p>
         <a class="card-header-icon">
-          <b-icon :icon="props.open ? 'chevron-up' : 'chevron-down'"> </b-icon>
+          <NeoIcon :icon="props.open ? 'chevron-up' : 'chevron-down'">
+          </NeoIcon>
```
Comment (roiLeo) [inline components/rmrk/Gallery/PriceChart.vue:17]:
```suggestion
          <NeoIcon :icon="props.open ? 'chevron-up' : 'chevron-down'" />
```

File: components/series/SeriesTable.vue [hunk 5]
```diff
@@ -302,7 +302,7 @@
           v-if="!isLoading"
           :to="`/${urlPrefix}/collection/${props.row.id}?tab=chart&locate=true`"
           target="_blank">
-          <b-icon icon="chart-line"> </b-icon>
+          <NeoIcon icon="chart-line"> </NeoIcon>
```
Comment (roiLeo) [inline components/series/SeriesTable.vue:305]:
```suggestion
          <NeoIcon icon="chart-line" />
```

File: components/series/SeriesTable.vue [hunk 6]
```diff
@@ -316,7 +316,7 @@
           v-if="!isLoading"
           :to="`/${urlPrefix}/collection/${props.row.id}?tab=history&locate=true`"
           target="_blank">
-          <b-icon icon="list-ul"> </b-icon>
+          <NeoIcon icon="list-ul"> </NeoIcon>
```
Comment (roiLeo) [inline components/series/SeriesTable.vue:319]:
```suggestion
          <NeoIcon icon="list-ul" />
```

File: components/shared/ScrollTopButton.vue [hunk 7]
```diff
@@ -3,7 +3,7 @@
     v-if="showBtn"
     class="scroll-top-button button is-justify-content-center"
     @click="scrollToTop">
-    <b-icon
+    <NeoIcon
       icon="chevron-up"
       size="is-small"
       :type="$colorMode.value === 'dark' ? 'is-white' : 'is-black'" />
```
Comment (roiLeo) [inline components/shared/ScrollTopButton.vue:9]: type prop won't work here, replace with class (maybe use `has-text-white` & `has-text-black`)

File: components/shared/collapse/DescriptionWrapper.vue [hunk 8]
```diff
@@ -9,7 +9,8 @@
     </div>
     <div class="has-text-centered">
       <a v-if="hasWrapper" @click.prevent="toggleDescription">
-        <b-icon :icon="activeWrapper ? 'chevron-down' : 'chevron-up'"></b-icon>
+        <NeoIcon
+          :icon="activeWrapper ? 'chevron-down' : 'chevron-up'"></NeoIcon>
```
Comment (roiLeo) [inline components/shared/collapse/DescriptionWrapper.vue:13]:
```suggestion
        <NeoIcon
          :icon="activeWrapper ? 'chevron-down' : 'chevron-up'" />
```

File: components/shared/gallery/BaseGalleryItem.vue [hunk 9]
```diff
@@ -63,23 +63,23 @@
           v-if="!isLoading && imageVisible"
           id="theatre-view"
           @click="toggleView">
-          <b-icon :icon="viewMode === 'default' ? 'image' : 'cube'"> </b-icon>
+          <NeoIcon :icon="viewMode === 'default' ? 'image' : 'cube'"> </NeoIcon>
```
Comment (roiLeo) [inline components/shared/gallery/BaseGalleryItem.vue:66]:
```suggestion
          <NeoIcon :icon="viewMode === 'default' ? 'image' : 'cube'" />
```

File: components/shared/gallery/BaseGalleryItem.vue [hunk 10]
```diff
@@ -63,23 +63,23 @@
...
         </button>
         <button
           v-if="!isLoading && imageVisible"
           id="fullscreen-view"
           :class="{ fullscreen: isFullScreenView }"
           @keyup.esc="minimize"
           @click="toggleFullScreen">
-          <b-icon :icon="isFullScreenView ? 'compress' : 'expand'"> </b-icon>
+          <NeoIcon :icon="isFullScreenView ? 'compress' : 'expand'"> </NeoIcon>
```
Comment (roiLeo) [inline components/shared/gallery/BaseGalleryItem.vue:74]:
```suggestion
          <NeoIcon :icon="isFullScreenView ? 'compress' : 'expand'" />
```

File: components/shared/gallery/BaseGalleryItem.vue [hunk 11]
```diff
@@ -63,23 +63,23 @@
...
         </button>
         <button
           v-if="!isLoading && imageVisible"
           id="tile-view"
           :class="{ tile: isTileView }"
           @click="toggleTileView"
           @keyup.esc="exitTileView">
-          <b-icon :icon="isTileView ? 'compress' : 'table'"> </b-icon>
+          <NeoIcon :icon="isTileView ? 'compress' : 'table'"> </NeoIcon>
```
Comment (roiLeo) [inline components/shared/gallery/BaseGalleryItem.vue:82]:
```suggestion
          <NeoIcon :icon="isTileView ? 'compress' : 'table'" />
```

File: components/unique/Gallery/Item/Detail.vue [hunk 12]
```diff
@@ -50,7 +50,7 @@
       <p class="label">
         {{ $t('delegate') }}
         <span>
-          <a class="has-text-danger"><b-icon icon="times"> </b-icon></a>
+          <a class="has-text-danger"><NeoIcon icon="times"> </NeoIcon></a>
```
Comment (roiLeo) [inline components/unique/Gallery/Item/Detail.vue:53]:
```suggestion
          <a class="has-text-danger"><NeoIcon icon="times" /></a>
```

File: pages/carbonless.vue [hunk 13]
```diff
@@ -149,3 +149,12 @@
...
 </template>
+
+<script lang="ts">
+import { Component, Vue } from 'nuxt-property-decorator'
+import { NeoIcon } from '@kodadot1/brick'
+@Component<Carbonless>({
+  components: { NeoIcon },
+})
+export default class Carbonless extends Vue {}
+</script>
```
Comment (roiLeo) [inline pages/carbonless.vue:160]: better to create with composition api
```suggestion
<script lang="ts" setup>
import { NeoIcon } from '@kodadot1/brick'
</script>
```

File: components/TheFooter.vue [hunk 14]
```diff
@@ -112,12 +120,11 @@
...
               </svg>
 
-              <b-icon
+              <NeoIcon
                 v-else
                 :pack="item.name == 'Swag' ? 'fas' : 'fab'"
                 :icon="item.icon"
-                size="is-small"
-                :type="isDarkMode ? 'is-white' : 'is-black'" />
+                size="is-small" />
```
Comment (roiLeo) [inline components/TheFooter.vue:127]: remove color? size?

File: components/explore/FilterMenuButton.vue [hunk 15]
```diff
@@ -5,7 +5,7 @@
         class="is-hidden-mobile"
         :class="{ disabled: disabled }"
         @click="toggleSidebarFilters">
-        <b-icon
+        <NeoIcon
           :icon="isSidebarFiltersOpen && !disabled ? 'times' : 'bars'"
           size="is-medium" />
```
Comment (roiLeo) [inline components/explore/FilterMenuButton.vue:10]:
```suggestion
          size="medium" />
```

File: components/identity/module/IdentityPopoverHeader.vue [hunk 16]
```diff
@@ -10,27 +10,29 @@
...
           icon="copy"
           size="is-small"
-          type="is-link is-clickable"
+          class="is-link is-clickable"
           data-cy="identity-clipboard"
           @click.native="toast('Copied to clipboard')" />
       </div>
       <a
-        v-if="identity?.twitter"
+        v-if="identity?.twitte"
```
Comment (roiLeo) [inline components/identity/module/IdentityPopoverHeader.vue:22]: wut?
```suggestion
        v-if="identity?.twitter"
```

File: components/identity/module/IdentityPopoverHeader.vue [hunk 17]
```diff
@@ -10,27 +10,29 @@
             {{ identity?.display || shortenedAddress }}</span
           >
         </nuxt-link>
-        <b-icon
+        <NeoIcon
           v-clipboard:copy="address"
           icon="copy"
           size="is-small"
```
Comment (roiLeo) [inline components/identity/module/IdentityPopoverHeader.vue:16]:
```suggestion
```

File: components/rmrk/Create/DropUpload.vue [hunk 18]
```diff
@@ -19,15 +19,15 @@
           @input="createInput">
           <section class="section">
             <div class="content has-text-centered">
-              <b-icon v-if="!file && !url" :icon="icon" size="is-large" />
+              <NeoIcon v-if="!file && !url" :icon="icon" size="is-large" />
```
Comment (roiLeo) [inline components/rmrk/Create/DropUpload.vue:22]:
```suggestion
              <NeoIcon v-if="!file && !url" :icon="icon" size="large" />
```

File: components/rmrk/Create/DropUpload.vue [hunk 19]
```diff
@@ -19,15 +19,15 @@
...
               <div v-if="url && !isModelMedia" @click.prevent>
                 <MediaResolver
                   :src="url"
                   :mime-type="mimeType"
                   :preview="false"
                   @error="hasError = true" />
               </div>
-              <b-icon v-if="hasError" icon="eye-slash" size="is-large" />
+              <NeoIcon v-if="hasError" icon="eye-slash" size="is-large" />
```
Comment (roiLeo) [inline components/rmrk/Create/DropUpload.vue:30]:
```suggestion
              <NeoIcon v-if="hasError" icon="eye-slash" size="large" />
```

File: components/search/SearchSuggestion.vue [hunk 20]
```diff
@@ -151,7 +151,7 @@
         class="is-flex is-align-items-center is-justify-content-space-between mb-1 search-history-item"
         @click="goToExploreResults(item)">
         <div class="is-flex is-align-items-center">
-          <b-icon icon="history" size="is-small" />
+          <NeoIcon icon="history" size="is-small" />
```
Comment (roiLeo) [inline components/search/SearchSuggestion.vue:154]:
```suggestion
          <NeoIcon icon="history" />
```

File: components/shared/ScrollTopButton.vue [hunk 21]
```diff
@@ -3,7 +3,7 @@
     v-if="showBtn"
     class="scroll-top-button button is-justify-content-center"
     @click="scrollToTop">
-    <b-icon
+    <NeoIcon
       icon="chevron-up"
       size="is-small"
```
Comment (roiLeo) [inline components/shared/ScrollTopButton.vue:8]:
```suggestion
```

File: components/shared/Sharing.vue [hunk 22]
```diff
@@ -63,36 +63,36 @@
...
+            <NeoIcon size="is-large" pack="fab" icon="whatsapp" />
           </ShareNetwork>
           <ShareNetwork
             tag="button"
             class="button share__button is-medium"
             network="email"
             :url="realworldFullPath"
             :title="$t(label)">
-            <b-icon size="is-large" pack="fas" icon="envelope" />
+            <NeoIcon size="is-large" pack="fas" icon="envelope" />
```
Comment (roiLeo) [inline components/shared/Sharing.vue:90]:
```suggestion
            <NeoIcon size="large" pack="fas" icon="envelope" />
```

**Review by roiLeo — COMMENTED** [review]
Some smoll stuff:
- self closing tag
- size prop
- default `small`? does it make big change on UI?

otherwise code lgtm ✅

**Review by roiLeo — COMMENTED** [review]
Some smoll stuff:
- self closing tag
- size prop
- default `small`? does it make big change on UI?

otherwise code lgtm ✅

**Comment by vikiival** [comment]
Agree with @roiLeo

**Comment by roiLeo** [comment]
> Agree with @roiLeo

**Comment by prachi00** [comment]
sp default medium was looking too big, small looks like what we had before

**Comment by daiagi** [comment]
> Also, does anyone know why deepsource says neoIcon is defined but isnt used, when it IS being used

got the same issue with NeoInput
https://github.com/kodadot/nft-gallery/pull/5765#issuecomment-1528647824

something is not right with deepSource config i guess

@kodadot/internal-dev  ideas?

**Review by roiLeo — APPROVED** [review]
> got the same issue with NeoInput [#5765 (comment)](https://github.com/kodadot/nft-gallery/pull/5765#issuecomment-1528647824)
> 
> something is not right with deepSource config i guess

No idea about that one, deepsource config hasn't been touched in a while

code lgtm ✅

File: components/common/IdentityForm.vue [hunk 23]
```diff
@@ -89,6 +89,7 @@ import { notificationTypes, showNotification } from '@/utils/notification'
 import { onApiConnect } from '@kodadot1/sub-api'
 import { hexToString, isHex } from '@polkadot/util'
 import { Data } from '@polkadot/types'
+import { NeoIcon } from '@kodadot1/brick'
 const Auth = defineAsyncComponent(() => import('@/components/shared/Auth.vue'))
```
Comment (vikiival) [inline components/common/IdentityForm.vue:93]:
```suggestion
import { NeoIcon } from '@kodadot1/brick'

const Auth = defineAsyncComponent(() => import('@/components/shared/Auth.vue'))
```

File: components/rmrk/Create/SimpleMint.vue [hunk 24]
```diff
@@ -224,7 +224,7 @@ import AuthMixin from '~/utils/mixins/authMixin'
 import { useFiatStore } from '@/stores/fiat'
 import { usePinningStore } from '@/stores/pinning'
 import { usePreferencesStore } from '@/stores/preferences'
-
+import { NeoIcon } from '@kodadot1/brick'
 const components = {
```
Comment (vikiival) [inline components/rmrk/Create/SimpleMint.vue:228]:
```suggestion
import { NeoIcon } from '@kodadot1/brick'

const components = {
```

File: components/rmrk/Gallery/EmotionList.vue [hunk 25]
```diff
@@ -32,7 +32,7 @@ import orderBy from 'lodash/orderBy'
 
 import { Emotion } from '../service/scheme'
 import EmotionModal from './EmotionModal.vue'
-
+import { NeoIcon } from '@kodadot1/brick'
 const issuerId = (emotion: Emotion) => emotion.caller
```
Comment (vikiival) [inline components/rmrk/Gallery/EmotionList.vue:36]:
```suggestion
import { NeoIcon } from '@kodadot1/brick'

const issuerId = (emotion: Emotion) => emotion.caller
```

**Comment by prachi00** [comment]
can we merge this?

File: components/common/IdentityForm.vue [hunk 26]
```diff
@@ -7,7 +7,7 @@
         <b-tooltip
           :label="$i18n.t('identity.fundsReserve')"
           position="is-bottom">
-          <b-icon icon="info-circle" />
+          <NeoIcon icon="info-circle" pack="fas" size="small" />
```
Comment (vikiival) [inline components/common/IdentityForm.vue:10]:
```suggestion
          <NeoIcon icon="info-circle" pack="fas" />
```

**Comment by vikiival** [comment]
Spawing @exezbcz

**Comment by exezbcz** [comment]
hi! some icons are broken, can you please fix them: 
### footer
image
- the hover bg color is oki but the icon should be k-black in case of dark mode, also it should not change when you hover over the icon itself. 
- canary:
- 
image
deploy (hover over button): 
image
deploy hover over icon: 
image


### Waiting for authorization 
canary: image
deploy: image

**Comment by prachi00** [comment]
> hi! some icons are broken, can you please fix them:
> 
> ### footer
> image
> 
> - the hover bg color is oki but the icon should be k-black in case of dark mode, also it should not change when you hover over the icon itself. - canary: -
> image
> 
> deploy (hover over button):
> image
> 
> deploy hover over icon:
> image
> 
> ### Waiting for authorization
> canary: image deploy: image

done @exezbcz

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
@prachi00 please resolve conflicts

**Comment by prachi00** [comment]
> @prachi00 please resolve conflicts

done

**Comment by vikiival** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 24.69 USD/KSM ~ 2.025 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0xcd09488e10a7d0e4af36cb690fe97c3f5736bc9ec5409e3f77a31698c463a760](https://kusama.subscan.io/extrinsic/0xcd09488e10a7d0e4af36cb690fe97c3f5736bc9ec5409e3f77a31698c463a760)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**